core/core-impl/src/test/java/ai/timefold/solver/core/impl/domain/solution/cloner/gizmo/GizmoSolutionClonerTest.java

```diff
@@ -69,7 +69,7 @@ protected <Solution_> SolutionCloner<Solution_> createSolutionCloner(SolutionDes
                             generateGizmoSolutionOrEntityDescriptor(solutionDescriptor, clazz));
                 });
 
-        GizmoSolutionClonerImplementor.defineClonerFor(classCreator, solutionDescriptor,
+        new GizmoSolutionClonerImplementor().defineClonerFor(classCreator, solutionDescriptor,
```

Comment:
Same comment as above; I think it could've been made static and hidden the instance.

Reply:
Issue is, Quarkus need to pass its own subclass as an instance instead of using the default one; I prefer `new Subclass().defineClonerFor(args)` to `Implementor.defineClonerFor(new Subclass(), args)`

Reply:
> Issue is, Quarkus need to pass its own subclass as an instance instead of using the default one; I prefer `new Subclass().defineClonerFor(args)` to `Implementor.defineClonerFor(new Subclass(), args)`

In the example you give, I can see how that is a question of personal preference.
But on line 72, where I put my comment, you're creating the instance without assigning it to any variable; why have the user create the instance then? It just looks odd.